store/instance.go

```diff
@@ -63,11 +63,12 @@ type instanceRaw struct {
 	ID int
 
 	// Standard fields
-	RowStatus api.RowStatus
-	CreatorID int
-	CreatedTs int64
-	UpdaterID int
-	UpdatedTs int64
+	RowStatus  api.RowStatus
+	CreatorID  int
+	CreatedTs  int64
+	UpdaterID  int
+	UpdatedTs  int64
+	ResourceID string
```

Comment:
Do we need to use another new struct and functions only used by the new OpenAPI? @d-bytebase

Reply:
Yup, let's do that.